src/components/PopUpWithForm.js

```diff
@@ -32,5 +51,11 @@ export default class PopUpWithForm extends PopUp {
     this.inputValues.forEach((input) => {
       input.value = data[input.name];
     });
+
+    close();
+    {
+      this._popup.querySelector(".modal__form").reset();
```

Comment:
Please, delete the `close` method from `PopUpWithForm` because you don't need it. Resetting should be in `then` rather than after closing